src/dmd/dsymbolsem.d

```diff
@@ -6625,7 +6625,23 @@ void aliasSemantic(AliasDeclaration ds, Scope* sc)
 
         if (e)  // Try to convert Expression to Dsymbol
         {
-            s = getDsymbol(e);
+            if (TupleExp te = e.isTupleExp())
+            {
+                Objects* objs = new Objects;
+                foreach (ex; *te.exps)
+                {
+                    if (Dsymbol sx = getDsymbol(ex))
+                        objs.push(sx);
+                    else break;
```

Comment:
the style here looks somewhat funky.

Reply:
But at least I reference properly an issue and the PR title is not a click bait. You know what it is about before clicking the link. in reference to #11585 :)